### DISCUSSION TITLE
Automatic MR creation with Dashboard enabled?

### DISCUSSION THREAD

**Discussion by lschaertel** [renovatebot/renovate#13931] [Request Help] [answered]
Hello,

I was wondering if it was possible to let renovate automatically create the MRs for my project, but still use the Issue Dashboard to have an overview of all the different dependencies/upgrades?
I was just able to enable the Dashboard and then manually approve the tasks I want to upgraded - But the combination, I was not really able to do so.

Thanks in advance

**Comment by rarkins** [comment, marked as answer]
What you describe is the default behaviour. Possibly if you only just installed the bot then you're seeing a limit of 2 MRs per hour, which is the default for config:base

**Reply by lschaertel** [reply]
Oh ok, that maybe could be the reason then.
Just to understand it correctly - If I use the default config `config:base`, I will have the dependency dashboard issue which shows all the open dependencies. And every time the schedule for one task is "valid", it will automatically then create the MRs for this task.
And if I dont want to wait for those schedules, I can still force the creation by manually approving a task on this issue?
